core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -295,11 +295,20 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with TimeLi
 
   override def beforeEach(): Unit = {
     super.beforeEach()
-    init(new SparkConf())
+    firstInit = true
+    setConf("spark.master" -> "local[2]", "spark.app.name" -> "DAGSchedulerSuite")
```

Comment:
We'd better to expose the `conf` via a function to set the conf. `setConf` is designed to be used by the test only.

Reply:
OK